indico/modules/events/timetable/controllers/manage.py

```diff
@@ -112,3 +113,21 @@ def _process_PATCH(self):
             with track_time_changes():
                 update_timetable_entry(self.timetable_entry, updates)
         return jsonify()
+
+    def _process_DELETE(self):
+        """Delete a timetable entry"""
+        delete_timetable_entry(self.timetable_entry)
+
+
+class RHTimetableBalloon(RHTimetableREST):
```

Comment:
I don't think subclassing this is a good idea. Getting the timetable entry is not that much code and the REST RH has nothing to do with this one.